content/post/087-deprecating-the-old-docs-site.md

```diff
@@ -0,0 +1,51 @@
+
+So with all that in mind, the IPFS docs and UX working group built a new site to host the documentation. The site build was finished towards the end of 2019, and a beta version was made available to the public. Call-outs were placed on the current IPFS docs site to encourage readers to test the new website. You can see a preview of the site at [docs-beta.ipfs.io](https://docs-beta.ipfs.io).
+
+## The deprecation plan
+
+We've built the site, put it online, and it's been well received. Next comes the tricky part: removing the old website and putting this one in its place.
+
+Back in the early days of the internet, deprecating a site was as simple as putting up a notice saying things were going to be down for an hour or so, deleting everything in `/var/www/html`, and replacing it with the new website. Sadly the good-old-days are gone, and we now live in a world full of checks, balances, and redirects. With that in mind, here is our plan to deprecate the old site and replace it with the new one.
+
+### Inform the world
+
+The first step is to let everyone know what we're going to be doing. This post serves as part of the notification process, along with call-outs on the legacy site, and in the `ipfs/docs` GitHub repository.
+
+### Plan for redirects
+
+There have been several changes to the structure and naming conversions in the docs repo. Significant URL redirects must be put in place from legacy links to where the content now lives. The idea here is to minimize the amount of 404 errors encountered by the reader.
+
+### Update external services
+
+Tools like Google Analytics and Bing Webmaster all need to be informed of the changes. Unfortunately, Alphabet's Google Search and Microsoft's bing are both necessary evils that enable us to reach as many users as possible. In the future, we'll be able to distance ourselves from centralized companies. Still, for now, we need to use them to help people access IPFS more easily.
+
+### Combine all the repos
+
+The IPFS docs have been stuck in limbo for a few months, with GitHub issues living in `ipfs/docs`, but the actual documentation and website content living in `ipfs/ipfs-docs-v2`. This has caused some confusion with posting issues in repo while reflecting those changes in another.
+
+Past of the deprecation place is to merge `ipfs/ipfs-docs-v2` into `ipfs/docs`. All the content in `ipfs/docs` will be overwritten, but the issues will remain intact. Open issues will have their links checked to make sure they are referencing the correct and updated pages.
+
+### Follow-up
+
+This final task will take place around 6-12 weeks after the site deprecation. There are three things the docs team will need to focus on at this stage:
+
+1. Remove the _this URL has changed_ notices in any existing pages.
+1. Re-crawl the site to make sure all the links and references are still working and up to date.
+1. Check the top 404 referrers, and create a fix for them where possible.
```

Comment:
```suggestion
1. Check the top 404 referrers, and create a fix for them where possible.
2. Search all content in the IPFS and Protocol Labs GitHub orgs for references to old site pages, and fix those (so we don't 404 ourselves!).
```
Might want to call out this specific instance of fixing broken links, since it'll probably be the biggest offender.